### PR TITLE
Update examples to centralize USE_SIMULATION in _constants.py

### LINKED ISSUE #341: `\examples\..\readme.md` Documentation Error
# Bug Report

`USE_SIMULATION = False` must be set in the `<inst>_helpers.py` file not the `measurement.py` to use a physical instrument.  

## Your Environment

* OS & Device: [Windows on PC]
* `ni-measurementlink-service` version [1.1]
* MeasurementLink version [2023Q3]
* Python version [3.11.4] Not using virtual environment

[AB#2463575](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2463575)

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Fixes #341. The README files were saying USE_SIMULATION is in measurement.py. It has been moved to _constants.py and references to it have been updates including documentation.

### Why should this Pull Request be merged?

Documentation accuracy and consistency among examples of having a _constants.py with USE_SIMULATION if needed.

### What testing has been done?

None

### CODE REVIEW

File: examples/nidcpower_source_dc_voltage/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
 This example requires an NI SMU that is supported by NI-DCPower (e.g. PXIe-4141).
 
 By default, this example uses a simulated instrument. To use a physical instrument, edit
-`measurement.py` and `teststand_fixture.py` to specify `USE_SIMULATION = False`.
+`_nidcpower_helpers.py` and `teststand_fixture.py` to specify `USE_SIMULATION = False`.
```
Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:35]: There is no USE_SIMULATION constant in [teststand_fixture.py](https://github.com/ni/measurementlink-python/blob/main/examples/nidcpower_source_dc_voltage/teststand_fixture.py).

Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:35]: I thought we were adding `_constants.py`?

Please don't customize `_nidcpower_helpers.py` and `_visa_helpers.py` for each example. Make them all the same so we can more easily keep them up to date.

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/nidcpower_source_dc_voltage/README.md (hunk 1 above)
Comment (bkeryan) [reply examples/nidcpower_source_dc_voltage/README.md:35]: This is in reference to https://github.com/ni/measurementlink-python/pull/323

Comment (dixonjoel) [reply examples/nidcpower_source_dc_voltage/README.md:35]: Ah got it. I'll follow that pattern for these examples then. Is the plan for _nidcpower_helpers.py and teststand_fixture.py both to reference _constants.py for USE_SIMULATION to single source it?

Comment (bkeryan) [reply examples/nidcpower_source_dc_voltage/README.md:35]: @Jayaseelan-James @samchris007

Comment (Jayaseelan-James) [reply examples/nidcpower_source_dc_voltage/README.md:35]: That's right, @dixonjoel. Going forward, in all the examples we'll have "USE_SIMULATION" in `_constants.py`  as a single source of truth.

Comment (dixonjoel) [reply examples/nidcpower_source_dc_voltage/README.md]: Added _constants.py where needed and reference USE_SIMULATION from there.

**Review by bkeryan — APPROVED** [review]

**Pull request merged**